demo/data.ts

```diff
@@ -209,3 +252,15 @@ export function generateData(seriesLength: number, includeMinMaxRange: boolean):
   }
   return results;
 }
+
+export function generateBubbleData(multiData = multi) {
+  return multiData.map(({name, series}) => ({
+    name,
```

Comment:
Are we using the name property anywhere except the tooltip?

I think the data format for this should be a single series, where:
`name` - color, legend
`x` - x axis
`y` - y axis
`r` - radius (maybe rename the property to `radius` so it's more explicit

and we should remove the multiSeries example.